Sources/OVP/KalturaOVPPlayer.swift

```diff
@@ -122,13 +263,11 @@ import PlayKitProviders
         ovpMediaProvider.set(sessionProvider: sessionProvider)
         
         ovpMediaProvider.loadMedia { [weak self] (pkMediaEntry, error) in
```

Comment:
If self is not used it can be removed.
```suggestion
        ovpMediaProvider.loadMedia { (pkMediaEntry, error) in
```